core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -228,6 +228,11 @@ private[spark] class SparkSubmit extends Logging {
     val childClasspath = new ArrayBuffer[String]()
     val sparkConf = args.toSparkConf()
     if (sparkConf.contains("spark.local.connect")) sparkConf.remove("spark.remote")
+    if (sparkConf.getBoolean(STRUCTURED_LOGGING_ENABLED.key, defaultValue = true)) {
```

Comment:
does this work for spark-shell as well? And thriftserver?

Reply:
Yes